staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -89,6 +139,35 @@ func (o *AuditLogOptions) ApplyTo(c *server.Config) error {
 		}
 	}
 	c.LegacyAuditWriter = w
-	c.AuditBackend = pluginlog.NewBackend(w)
+
+	if advancedAuditingEnabled() {
+		c.AuditBackend = appendBackend(c.AuditBackend, pluginlog.NewBackend(w))
+	}
+	return nil
+}
+
+func (o *AuditWebhookOptions) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&o.ConfigFile, "audit-webhook-config-file", o.ConfigFile,
+		"Path to a kubeconfig formatted file that defines the audit webhook configuration."+
+			" Requires the 'AdvancedAuditing' feature gate.")
+	fs.StringVar(&o.Mode, "audit-webhook-mode", o.Mode,
+		"If sending audit events should block server responses or if events should be"+
```

Comment:
am no native speaker, is this "If ... should" construct right?

And guar{e->a}ntees.